### PR TITLE
Hide deprecated warnings

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/437

Note: cms3 travis failures are expected

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ language: php
 os: linux
 
 # Using xenial which will use MySQL 5.7 by default
```
Comment (dhensby) [inline .travis.yml]: Perhaps the comment should be updated or removed?

**Review by dhensby — COMMENTED** [review]
What is the advantage of moving the error reporting from an ini file to the `_ss_enviornment.php` file? It feels like moving it to the php file is going to cause future problems if it gets retired.

Updating the ini file the way we are does appear to be the recommended way: https://docs.travis-ci.com/user/languages/php/#Custom-PHP-configuration

**Comment by emteknetnz** [comment]
> It feels like moving it to the php file is going to cause future problems if it gets retired.

I think the chance of _ss_environment.php getting retired from CMS 3 projects it's pretty low :-)

> What is the advantage of moving the error reporting from an ini file to the _ss_enviornment.php

Possibly none other then we don't need to spend any time getting it to work with via ini.  Silverstripe sets some `error_reporting()` of its own so there may be a conflect.  I don't think it really matters how we set it in the context of a travis build

**Comment by michalkleiner** [comment]
Since the config was `E_ALL & ~E_DEPRECATED` already and this change just moves it around, in the context of the issue title, is it even needed? Does updating the dist that travis runs make any difference to that?

**Comment by emteknetnz** [comment]
Config wasn't working for some reason and travis was spamming deprecated warnings so I changed around how it was set and it resolved

> Does updating the dist that travis runs make any difference to that?

Probably not, though bionic is what we run on the [travis shared config](https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/base.yml#L5)

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
This failure should be pretty easy to fix. Just need to install another PHP extension.
> Failed asserting that 'MIME validation failed: Could not find a MIME type for extension php' contains "Extension is not allowed".

All the other failures are pre-existing, so we can live with them.

**Comment by emteknetnz** [comment]
That's a pre-existing failure present on the [PHP 7.3 build](https://travis-ci.com/github/silverstripe/cwp-recipe-basic/jobs/508791682#L1005)

We're specifically not fixing pre-existing failures as part of this issue, we're simply checking that php7.3 and php7.4 behave the same - see the first AC on https://github.com/silverstripeltd/product-issues/issues/437

File: .travis.yml (hunk 1 above)
Comment (maxime-rainville) [reply .travis.yml]: @emteknetnz Maybe blast this comment.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Discussed the MIME type failure and decided to live with it. Just need to address Dan's comment.

**Comment by emteknetnz** [comment]
@maxime-rainville have removed comment

**Review by maxime-rainville — APPROVED** [review]

**Pull request merged**